### ISSUE TITLE
switch non-ascii unicode chars to images

### ISSUE THREAD

**Issue by yongrenjie** [alan-turing-institute/demoland-web#31]
triangles in collapsible, as well as the swap button

**Comment by martinfleis** [comment]
Is this misalignment of X within the rectangle part of the issue?
![Screenshot 2023-06-27 at 16 00 55](https://github.com/alan-turing-institute/demoland-web/assets/36797143/a71830a5-cd5d-42d9-ac6d-d87c68d98c93)

**Comment by yongrenjie** [comment]
No, not the same, that's just another weird Safari rendering bug where it shifted part of the SVG downwards but not another part of it!

I fixed it earlier this afternoon: https://github.com/alan-turing-institute/demoland-web/pull/30/commits/55f0cdc5befa304bf7aebb74ad448b57fa921999
